components/automate-gateway/gateway/services.go

```diff
@@ -113,6 +115,18 @@ func (s *Server) RegisterGRPCServices(grpcServer *grpc.Server) error {
 		trialLicenseURL,
 	))
 
+	if os.Getenv("CHEF_DEV_ENVIRONMENT") == "true" {
+		cdsClient, err := clients.CdsClient()
+		if err != nil {
+			log.WithFields(log.Fields{
+				"service": "automate-cds",
+				"error":   err,
+			}).Fatal("Could not create client")
+		}
+
+		pb_cds.RegisterCdsServer(grpcServer, handler.NewCdsServer(cdsClient))
+	}
+
```

Comment:
Since the automate-cds service only starts in the hab studio dev environment, we do not want to try to connect to the service in none dev. 

This will be removed when we want users to use the automate-cds.